utility/utils.py

```diff
@@ -557,6 +561,16 @@ def email_results(results_list, run_id, trigger_user, run_dir, suite_run_time, s
     sender = "cephci@redhat.com"
     recipients = []
     address = cfg.get('address')
+    send_to_cephci = email_result.get("send_to_cephci", False)
+    try:
+        run_id = email_result["run_id"]
+        results_list = email_result["result"]
+        trigger_user = email_result["trigger_user"]
+        run_dir = email_result["run_directory"]
+        suite_run_time = email_result['total_time']
+        info = email_result['info']
+    except KeyError as kerr:
+        log.error(f"Key not found : {kerr}")
```

Comment:
Is it `fatal`? If yes, please exit and don't proceed further due to lack of information. Alternatively, we can chose to use `.get` if it is acceptable not to have those values.

Reply:
added a exit call with 1